docs/nginx.md

```diff
@@ -109,10 +115,11 @@ server {
     proxy_read_timeout                  900;
   }
 }
+}
 EOF
 
 # Now, create a password file for "testuser" and "testpassword"
-docker run --entrypoint htpasswd httpd:2.4 -bn testuser testpassword > auth/nginx.htpasswd
+docker run --rm --entrypoint htpasswd  registry:2 -bn testuser testpassword > auth/nginx.htpasswd
```

Comment:
We have already pulled the registry2 image and that (recently) includes the htpasswd tool, so why force the download of yet another image, https, which may not yet be on the docker server?